spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -116,6 +116,8 @@ module MiqAeServiceSpec
           Datacenter
           ManageIQ::Providers::BaseManager
           ManageIQ::Providers::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::Provider
```

Comment:
Thanks @gmcculloug, slowly but surly we're building up a corpus of info of what one needs to build a new provider from scratch.